psi4/CMakeLists.txt

```diff
@@ -157,6 +157,20 @@ get_property(_loc TARGET gau2grid::gg PROPERTY LOCATION)
 list(APPEND _addons ${_loc})
 message(STATUS "${Cyan}Using gau2grid${ColourReset}: ${_loc} (version ${gau2grid_VERSION})")
 
+if(${ENABLE_gauxc})
+    if((NOT "${gauxc_DIR}" STREQUAL "") AND "${IntegratorXX_DIR}" STREQUAL "")
+       set(IntegratorXX_DIR ${gauxc_DIR}/../IntegratorXX)
+    endif()
+
+    find_package(ExchCXX CONFIG REQUIRED PATHS ${gauxc_DIR}/../ExchCXX) 
```

Comment:
GauXC will handle the ExchCXX discovery, even in the case of a separate installation, no reason to discover it twice

Reply:
Yeah, I put this in because of Psi4's whole thing about discovering packages twice, but the recent build changes should cover this issue as well.